k8s/jiva/README.md

```diff
@@ -0,0 +1,124 @@
+This document will help to delete the auto-generated snapshots created when a Jiva replica restart happened or new replica got added to the Jiva controller. The steps for deleting the auto-generated snapshots  are the following:
+
+Get the details of Jiva controller pod using the following command.
+
+```
+kubectl get pod -n <PVC_namespace>
+```
+Example: 
+```
+kubectl get pod
+```
+Example Output:
+```
+NAME                                                            READY   STATUS    RESTARTS   AGE
+percona-7b64956695-kd9q4                                        1/1     Running   0          105s
+pvc-d01e90d9-a921-11e9-93c2-42010a8000ab-ctrl-df9c749cf-jp6mg   2/2     Running   0          104s
+pvc-d01e90d9-a921-11e9-93c2-42010a8000ab-rep-795d8c5cb8-4wb9c   1/1     Running   0          101s
+pvc-d01e90d9-a921-11e9-93c2-42010a8000ab-rep-795d8c5cb8-gfxg9   1/1     Running   0          101s
+pvc-d01e90d9-a921-11e9-93c2-42010a8000ab-rep-795d8c5cb8-n4cfq   1/1     Running   1          101s
+```
+
+Login to the Jiva controller pod using the following command. Use namespace where Jiva pods are running.
+```
+kubectl exec -it <jiva_controller_pod> -n <namespace> bash 
```

Comment:
Check if you can use: `kubectl exec -it <jiva_controller_pod> -n <namespace> -- jivactl snapshot ls` 
Will reduce some steps.

Reply:
Done.